theme/base-2016/partials/_footer.twig

```diff
@@ -17,6 +17,8 @@
 
 </footer>
 
-<script src="{{ asset('js/jquery.min.js', 'theme') }}"></script>
+{# Extensions and copypasta gleaned from other sources tend to cause multiple jQuery
+   includes. We only include it here conditionally, if not done so already. #}
+<script>window.jQuery || document.write("<script src='{{ asset('js/jquery.min.js', 'theme') }}'><\/script>");</script>
```

Comment:
Well it also says

>The document.write() is in a top level document. The intervention does not apply to document.written scripts within iframes

Which this is a top level doc

>the script in the document.write() is parser-blocking. Scripts with the 'async' or 'defer' attributes will still execute.

Which this is a parser blocking script


Also all that is superceded when a user is on a slow connection (and eventually slow wifi) it'll be removed and not execute


I just wanted to let you know thats all :)